MumblePAHelper.pro

```diff
@@ -19,4 +18,14 @@ HEADERS  += mumblepahelper.h \
 
 FORMS    += mumblepahelper.ui
 
-CONFIG += c++11
\ No newline at end of file
+win32:RC_ICONS += resources/MumblePAHelper.ico
+
+CONFIG += c++11
+
+Release: DESTDIR = $$_PRO_FILE_PWD_/release
```

Comment:
Have you tried using the pattern that Mumble itself uses?
It does:

	CONFIG(debug, debug|release) {
	  CONFIG += console
	  QMAKE_LIBDIR += ../../debug
	  DESTDIR	= ../../debug
	}

	CONFIG(release, debug|release) {
	  QMAKE_LIBDIR += ../../release
	  DESTDIR	= ../../release
	}